infrastructure/ai/llm_client.py

```
@@ -20,11 +21,20 @@ def __init__(self, provider=None):
         if provider:
             self.provider = provider
         else:
-            # 自动检测 API key 并选择提供者
-            api_key = self._get_api_key()
-            if api_key:
+            # 优先 ARK (OpenAI 兼容)，其次 Anthropic，最后 Mock
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)